plugins/@grouparoo/bigquery/src/lib/query-import/connection.ts

```diff
@@ -6,6 +6,7 @@ export function getConnection() {
   return buildConnection({
     app: "bigquery",
     name: "bigquery-query-import",
```

Comment:
you moved the others to have `export` before. SHoudl this be `bigquery-import-query`
fwiw, I like it how it is now.

Reply:
good point! 970702bd3bc1f30fc30ea26d3f39b91bc1e4ab69 changes all the import names too so all the name match the pattern `${app}-${direction}-${model}`